.htaccess.dist

```diff
@@ -31,3 +31,7 @@
   SetEnv DATABASE_PASSWORD password_example
 
 </IfModule>
+
+<IfModule mod_env.c>
```

Comment:
Yes may be better to put all the setEnv here

Reply:
I moved in another commit all setEnv variables in the mod_env module